packages/serve/src/startDevServer.ts

```diff
@@ -36,9 +36,9 @@ export default function startDevServer(compiler, cliOptions): object[] {
     const usedPorts: number[] = [];
     devServerOptions.forEach((devServerOpts): void => {
         const options = mergeOptions(cliOptions, devServerOpts);
-        options.host = options.host || 'localhost';
-        // devSever v4 handles the default port itself
+        // devSever v4 handles the default host and port itself
```

Comment:
I think we can break compatibility with webpack-dev-server@3, I think we need do it by default for `else` branch

Reply:
`isDevServer4` is negated in the condition.

Reply:
I definitely need a vacation